### ISSUE TITLE
Add a simple way to share a skunk report by setting an environment variable (SHARE=true)

### ISSUE THREAD

**Issue by etagwerker** [fastruby/skunk#52]
**Before we start...:**

- [x] I checked the documentation and didn't find this feature
- [x] I checked to make sure that this feature has not already been requested

**Branch/Commit:**

`main` branch

**Problem:**

There is no easy way to share Skunk reports with my team. I usually have to copy/paste data.

**Solution:**

> **As a developer using `skunk`**
> **I want to share Skunk data using https://skunk.fastruby.io**
> **So that I can quickly collaborate with my team**

It could work like this: 

```
SHARE=true bundle exec skunk
running flay smells

running flog smells
.............
running reek smells
.............
running complexity
.............
running attributes
.............
running churn
.............
running simple_cov
.............
New critique at file:////Users/etagwerker/Projects/fastruby/skunk/tmp/rubycritic/overview.html
...

SkunkScore Total: 612.31
Modules Analysed: 13
SkunkScore Average: 0.47100769230769230769230769231e2
Worst SkunkScore: 166.44 (lib/skunk/cli/commands/default.rb)
Shared at: https://skunk.fastruby.io/1335ce1
```

**Mockups:**

This is similar to the way that [`benchmark-ips`](https://github.com/evanphx/benchmark-ips#online-sharing) works: 

```
$ SHARE_URL=https://ips.fastruby.io ruby benchmarks/struct_vs_hash.rb
Ruby version: 2.6.3
Warming up --------------------------------------
              struct    29.961k i/100ms
                hash    28.806k i/100ms
Calculating -------------------------------------
              struct    302.729k (± 4.0%) i/s -      1.528M in   5.056211s
                hash    277.456k (± 3.9%) i/s -      1.411M in   5.095923s

Comparison:
              struct:   302728.7 i/s
                hash:   277456.0 i/s - 1.09x  (± 0.00) slower

Shared at: https://ips.fastruby.io/5
```

This is what it looks like for `benchmark.fyi` -> https://ips.fastruby.io/5

**Resources:**

There is already a `skunk.fyi` project: https://github.com/fastruby/skunk.fyi

It is currently running here: https://skunk.fastruby.io

- [x] I will abide by the [code of conduct](https://github.com/fastruby/skunk/blob/main/CODE_OF_CONDUCT.md)

**Comment by etagwerker** [comment]
This has been implemented in #56. The one thing that is missing is the documentation in the README: 

- [ ] Implement documentation for the share feature.

**Comment by etagwerker** [comment]
This was solved by #56
